cirq-core/cirq/sim/clifford/clifford_simulator_test.py

```diff
@@ -396,6 +394,7 @@ def random_clifford_gate():
     np.testing.assert_almost_equal(
         clifford_simulator.simulate(circuit).final_state.state_vector(),
         state_vector_simulator.simulate(circuit).final_state_vector,
+        6
```

Comment:
Nit: `decimal=6`

Reply:
Hi. I made the formatting change and committed it.